metadata/SSCWeb.json

```diff
@@ -11,7 +11,11 @@
         "command": "$NODEEXE $HAPISERVERPATH/bin/SSCWeb.js --id ${id} --parameters ${parameters} --start ${start} --stop ${stop}",
         "formats": ["csv"],
         "contact": "rweigel@gmu.edu",
-        "test": "$NODEEXE $HAPISERVERPATH/bin/SSCWeb.js --id ace --parameters Time --start 2000-01-01T00:00:00.000Z --stop 2000-01-01T23:59:59.000Z"
-    },
-    "catalog": "cd $HAPISERVERPATH/metadata/SSCWeb; $NODEEXE SSCWeb2HAPI.js"
+		"test": "$NODEEXE $HAPISERVERPATH/bin/SSCWeb.js --id ace --parameters Time --start 2000-01-01T00:00:00.000Z --stop 2000-01-01T23:59:59.000Z",
+		"testcommands": [
```

Comment:
To generate a catalog file in advance, before we link it below in the catalog section.